recipes/grid-strategy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "grid-strategy" %}
+{% set version = "0.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d8db1c12e6f33eb55ba56a5b19f4848aae862a9a343fea422a353c46ab6ccc23
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib
```

Comment:
matplotlib-base

Reply:
Ahh, this was created by grayskull, I'll also add an issue upstream.